netgrasp/netgrasp.py

```diff
@@ -10,17 +10,19 @@
 import logging
 import logging.handlers
 import pwd
-import sys
 import os
 import datetime
 import time
-import traceback
 
 netgrasp_instance = None
 
 BROADCAST = 'ff:ff:ff:ff:ff:ff'
 
-ALERT_TYPES = ['requested_ip', 'first_requested_ip', 'first_requested_ip_recently', 'seen_device', 'first_seen_device', 'first_seen_device_recently', 'seen_mac', 'first_seen_mac', 'seen_ip', 'first_seen_ip', 'seen_host', 'first_seen_host', 'seen_vendor', 'first_seen_vendor', 'device_stale', 'request_stale', 'changed_ip', 'duplicate_ip', 'duplicate_mac', 'network_scan', 'ip_not_on_network', 'src_mac_broadcast', 'requested_self']
+ALERT_TYPES = ['requested_ip', 'first_requested_ip', 'first_requested_ip_recently', 'seen_device', 'first_seen_device',
+               'first_seen_device_recently', 'seen_mac', 'first_seen_mac', 'seen_ip', 'first_seen_ip', 'seen_host',
+               'first_seen_host', 'seen_vendor', 'first_seen_vendor', 'device_stale', 'request_stale', 'changed_ip',
+               'duplicate_ip', 'duplicate_mac', 'network_scan', 'ip_not_on_network', 'src_mac_broadcast',
+               'requested_self']
```

Comment:
In the past I've left long lines (>120), but after manually wrapping and working on the code later, I appreciated being able to see all values/code on the page without the editor horizontally scrolling.

Above all I'd say it improves readability. I'll explain more in response to your comment below regarding the "if" logic manual wrap for "ng.email", since it's a better example as to why.